src/metrickit.cs

```diff
@@ -38,6 +38,10 @@ interface MXMetric : NSSecureCoding {
 		[MacCatalyst (14,0)]
 		[Export ("dictionaryRepresentation")]
 		NSDictionary _DictionaryRepresentation14 { get; }
+
+		[NoMac, iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+		[Field ("MXErrorDomain")]
+		NSString ErrorDomain { get; }
```

Comment:
This is not needed. It is already generated as part of the enum since you added `[ErrorDomain ("MXErrorDomain")]`

```suggestion
```

Reply:
Easy fix, committed so we do not wait on this little thing.